### ISSUE TITLE
Performance: Debian build runs slow asserts

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#519]
- [x] #508
- [x] slow asserts in lexer::parse_identifier (e.g. `look_up_in_unicode_table` is 2.65% in a CPU profile)

**Comment by strager** [comment]
Here's the CMake configure command run by the Debian build:
```
	cd obj-x86_64-linux-gnu && cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_AUTOGEN_VERBOSE=ON -DCMAKE_C_COMPILER=gcc-9 -DCMAKE_CXX_COMPILER=g\+\+-9 -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DQUICK_LINT_JS_INSTALL_VIM_DIR=share/vim/addons ..
```

**Comment by strager** [comment]
Fixed in commit 3090f7dc62701ea573f7682b52e407f30666173e.
